### ISSUE TITLE
Replace the use of @import

### ISSUE THREAD

**Issue by jtommy** [oruga-ui/theme-bulma#48]
According to https://sass-lang.com/blog/the-module-system-is-launched we should replace `@import` that has been deprecated since last October and we'll be removed before the end of year.

https://sass-lang.com/documentation/at-rules/import

**Comment by Tragio** [comment]
@jtommy to give you a quick reminder 😃

**Comment by blm768** [comment]
Since Bulma itself uses `@import` fairly heavily, this might end up blocked on the Bulma v1 release.

**Comment by messenjer** [comment]
Hello,

For the migration to bulma v1, I created a new PR using `@import` https://github.com/oruga-ui/theme-bulma/pull/101 

@Tragio, @blm768 may be you can test it.

Regards

Mathieu
